### DISCUSSION TITLE
How to show number of hidden files and assign a highlight color for them?

### DISCUSSION THREAD

**Discussion by hopezh** [nvim-tree/nvim-tree.lua#2223] [Q&A] [answered]
May I ask if there is a way to show number of hidden files in the current folder? 

Neo-tree seems to have such features: 

![image](https://github.com/nvim-tree/nvim-tree.lua/assets/23435810/5ae2409d-8bdc-49c9-92ad-4f6bfb59def3)

![image](https://github.com/nvim-tree/nvim-tree.lua/assets/23435810/afac0c74-eb4c-45a8-8e78-40e93df654d6)

**Comment by gegoune** [comment, marked as answer]
There is no such functionality at the moment but it would be good to have.

**Reply by alex-courtis** [reply]
It would be useful. It could be individual counts for No Buffer/Git Clean/Dotfiles/Git Ignore/Custom or one count for all hidden.

**Reply by hopezh** [reply]
Noted with thanks.

**Comment by alex-courtis** [comment]
See #2225
